Cargo.toml

```diff
@@ -16,6 +16,7 @@ members = [
     'pallets/parentchain',
     'pallets/test-utils',
     'pallets/group',
+    'pallets/bitacross-pallet',
```

Comment:
Traditionally the pallet is not suffixed with `-pallet` anymore, but that's a minor thing

Reply:
Will address in follow up PR